sig-service-catalog/charter.md

```diff
@@ -5,28 +5,28 @@ README](https://github.com/kubernetes/community/blob/master/committee-steering/g
 
 ## Scope
 
-Service Catalog is a Kubernetes extension project that implements the [Open
-Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It enables
-application developers to provision cloud services from within Kubernetes and
-integrates configuration and discovery of those services into Kubernetes
-resources.
+Service Catalog is a Kubernetes extension project that implements and provides
+specifications for kubernetes-specific solutions that enable application developers to
+consume cloud-provided or self-hosted services from within Kubernetes and integrates
+configuration and discovery of those services into Kubernetes resources.
```

Comment:
Is there a link to a discussion or issue with context for this change?

Reply:
Would also be interested in more context on this - Is the intent to drop support for the OSBAPI? Is there a different initiative that is moving forward?

Reply:
Not necessarily drop OSBAPI support. The SIG already adopted the minibroker subproject, and is looking to adopt the service binding spec https://github.com/kubernetes-sigs/service-catalog/issues/2857. We're looking to extend the scope so that we will be able to curate other technologies related to providing access to and controlling services aka stateful workloads.